examples/data/scripts/auth.py

```diff
@@ -3,6 +3,13 @@
 from __future__ import print_function
 
 import os
+try:
+    import pygtkcompat
+except ImportError:
+    pygtkcompat = None
+if not (pygtkcompat is None):
```

Comment:
You got me. This is where I deviated from the guide. I already forgot ‘is not’ is not ‘is’ ‘not’ in Python.
Don’t you think the less Pythonic way is more readable for those who don’t know ‘is not’ is parsed as a single operator?
(Yes, I know that’s a weak argument. Python code is supposed to be read by those who know Python, isn’t it?)